src/order_cmd.cpp

```diff
@@ -831,6 +831,14 @@ CommandCost CmdInsertOrder(DoCommandFlag flags, VehicleID veh, VehicleOrderID se
 					break;
 				}
 
+				case VEH_ROAD: {
+					if (!(wp->facilities & FACIL_BUS_STOP) || !(wp->facilities & FACIL_TRUCK_STOP)) return CommandCost(STR_ERROR_CAN_T_ADD_ORDER, STR_ERROR_NO_ROAD_WAYPOINT);
```

Comment:
What's going on here? A waypoint can't only be for trucks or buses...can it?

Reply:
Road waypoints have the FACIL_BUS_STOP and FACIL_TRUCK_STOP bits set.

Reply:
Is there not a better way to check if a tile is a road waypoint? This just feels really odd to me.

Reply:
This is in symmetry with and works the same way as the rail waypoint case directly above it and the ship case directly below it.

Reply:
Weird, but works for me. 🤷 Thanks for putting up with my silly questions. 😀